src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -355,6 +364,17 @@ export const selectDonationAmountFiat = createSelector(
   },
 )
 
+export const selectDonationAmountUsd = createSelector(
```

Comment:
This one may actually make sense to consume the existing selector not to duplicate complexity

Reply:
3ec6925cd